lsp-mode.el

```diff
@@ -57,6 +57,7 @@
 (require 'minibuffer)
 (require 'yasnippet nil t)
 (require 'lsp-protocol)
+(require 'page-break-lines)
```

Comment:
Is this built-in package? Else you have to put that one into package requires.

Reply:
true, also on makefile for CI tests I think

Reply:
I thought that it is built-in, it is in melpa and built-in for spacemacs.  I will look for a solution